udm/intermediate/shopify-samples/orders-json/Backorder BOPIS - content.json

```diff
@@ -0,0 +1,391 @@
+{
+  "order": {
+    "id": 5524199899389,
+    "admin_graphql_api_id": "gid://shopify/Order/5524199899389",
+    "app_id": 580111,
+    "browser_ip": "111.118.250.178",
+    "buyer_accepts_marketing": false,
+    "cancel_reason": null,
+    "cancelled_at": null,
+    "cart_token": "c1-205acf291edff427599b826e4816ad86",
+    "checkout_id": 34877545906429,
+    "checkout_token": "3a118dc38d9f924f60fdb5e7350f9923",
+    "client_details": {
+      "accept_language": "en-IN",
+      "browser_height": null,
+      "browser_ip": "111.118.250.178",
+      "browser_width": null,
+      "session_hash": null,
+      "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
+    },
+    "closed_at": null,
+    "company": null,
+    "confirmed": true,
+    "contact_email": "muskan@hotwax.com",
```

Comment:
use example.com email

Reply:
@sanjanashrivastava Updated the email in all json